app/partials/analysis/charts/tab-head.jade

```diff
@@ -0,0 +1,8 @@
+
+
+//Top of graph tabs
```

Comment:
(style): please always put a space before comment text, it helps with readability